src/core/hle/D3D8/Direct3D9/Direct3D9.cpp

```diff
@@ -2730,24 +2731,42 @@ void GetScreenScaleFactors(float& scaleX, float& scaleY) {
 	// The title uses MSAA, which increases the rendertarget size, but leaves the screen scale unaffected
 	// The backbuffer scale is (0.75, 1), which contributes to the screen scale
 	// So the Xbox expects vertices in 480*480 coordinate space
-
-	// SSAA increases the screen scale (but MSAA does not)
-	bool isMultiSampleEnabled = XboxRenderStates.GetXboxRenderState(xbox::X_D3DRS_MULTISAMPLEANTIALIAS);
+	// todo: find supersample type info from kelvin and use it here.
 	bool isSuperSampleMode = g_Xbox_MultiSampleType & xbox::X_D3DMULTISAMPLE_SAMPLING_SUPER;
-
-	// Apply multisample scale if supersampling is enabled
-	// Test cases:
-	// Antialias sample (text overlay is drawn with antialiasing disabled)
-	if (isMultiSampleEnabled && isSuperSampleMode) {
-		GetMultiSampleScaleRaw(scaleX, scaleY);
+	extern bool is_pgraph_using_NV2A_Kelvin(void);
+	extern void CxbxrGetSuperSampleScaleXY(float& x, float& y);
+
+	if (is_pgraph_using_NV2A_Kelvin()) {
+		bool isMultiSampleEnabled = NV2ARenderStates.GetXboxRenderState(xbox::X_D3DRS_MULTISAMPLEANTIALIAS);
+
+		// Apply multisample scale if supersampling is enabled
+		// Test cases:
+		// Antialias sample (text overlay is drawn with antialiasing disabled)
+		if (isMultiSampleEnabled && isSuperSampleMode) {
+			CxbxrGetSuperSampleScaleXY(scaleX, scaleY);
+			scaleX*= g_Xbox_BackbufferScaleX;
+			scaleY*= g_Xbox_BackbufferScaleY;
```

Comment:
```suggestion
			scaleX *= g_Xbox_BackbufferScaleX;
			scaleY *= g_Xbox_BackbufferScaleY;
```
Missing space between variable and multiplier assignment operator.